av1-rtp-spec.md

```diff
@@ -768,6 +768,20 @@ read_bit() {
 </code></pre>
 
 **ns(n)** - non-symmetric unsigned encoded integer with maximum number of values n (i.e., output in range 0..n-1).
+
+This descriptor is similar to ceiling of the base 2 logarithm of the input n, but reduces wastage incurred when encoding non-power of two value ranges by encoding 1 fewer bits for the lower part of the value range. For example, when n is equal to 5, the encodings are as follows (full binary encodings are also presented for comparison):

```

Comment:
Nit: for small number we spell them out, please change 2, 1 and 5 to 'two', 'one' and 'five'.

Reply:
I only changed the second 1 and 5, I think `base 2` is more readable than `base two` as a mathematical function

Reply:
I think you are right, LGTM!